packages/cluster-ui/README.md

```diff
@@ -17,3 +17,68 @@ export default props => (
   </div>
 );
 ```
+# Development Workflow with CRDB or CockroachCloud
+
+If you would like changes in this package to be reflected in a locally running DB Console instance, you will need to locally link the library to your DB Console project like this:
+
+In `packages/cluster-ui` run:
+
+```
+> yarn
+> yarn link
+> yarn run build:watch
+```
+
+Then in `pkg/ui` in CRDB repo run:
+
+```
+> yarn link "@cockroachlabs/cluster-ui"
+> yarn
```

Comment:
Is this second `yarn` command necessary for linking?

Reply:
maybe not...ill remove and see if someone complains :) the `make ui-watch` should run yarn anyway.